.github/workflows/artifact-policy-deploy.yml

```diff
@@ -0,0 +1,33 @@
+name: Deploy Artifact Registry Policy
+
+on:
+  push:
+    paths:
+      - artifact-registry-policy.json
+      - .github/workflows/artifact-policy-deploy.yml
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v3
+    
+    - name: Set up Google Cloud authentication
+      uses: google-github-actions/auth@v1
+      with:
+        credentials_json: ${{ secrets.GCP_SERVICE_ACCOUNT_KEY }}
+
+    - name: Install gcloud CLI
+      uses: google-github-actions/setup-gcloud@v1
+      with:
+        version: 'latest'
+
+    # Might have to change this policy to include a universal prefix if it doesn't work
+    - name: Set cleanup policy for Artifact Registry
+      run: |
+        gcloud artifacts repositories set-cleanup-policies pulse-backend \
+          --project=${{ secrets.GOOGLE_CLOUD_PROJECT }} \
+          --location=us-east4 \
```

Comment:
I think we have a variable called `REGION` in our GitHub actions secrets if you want to use that! (although I know we're planning to switch to Google credentials)

Reply:
changed to region for now, good call, but will probably change to google secrets later this week